coderd/coderdtest/coderdtest.go

```diff
@@ -433,7 +434,12 @@ func NewOptions(t testing.TB, options *Options) (func(http.Handler), context.Can
 	} else {
 		srv.Start()
 	}
-	t.Cleanup(srv.Close)
+	t.Logf("coderdtest server started on %s", srv.URL)
```

Comment:
I'm fairly sure `srv.Listener().Addr().String()` should equate to the same thing, but may be no harm to keep it explicitly the same.

Reply:
The URL contains the protocol, e.g. http vs https, which might be relevant. I kept them two different logs because the actual TCP listener is created on `NewUnstartedServer`, and then the server started later.